app/Http/Controllers/WebhookController.php

```diff
@@ -85,6 +85,30 @@ public function handle(Request $request) {
 
         $to = User::where(['active' => true, 'notify' => true])->pluck('email');
         $cc = env('ACTION_CC', false);
+        
+        if (env('FORWARD_WEBHOOK')) {
+            try {
+                $client = new Client();
+                $client->post(rtrim(env('FORWARD_WEBHOOK'), '/') . '/webhook', [
```

Comment:
Could we remove the `/webhook` requirement in the URL, and instead rely on it being set within `FORWARD_WEBHOOK`? I can see this being too restrictive to be useful in other use cases otherwise

Reply:
Good catch. Updated it just to use the value in .env as-is :+1: